src/support/rand.c

```diff
@@ -72,8 +76,29 @@ __wt_random_init_seed(WT_SESSION_IMPL *session, WT_RAND_STATE volatile *rnd_stat
     WT_RAND_STATE rnd;
 
     __wt_epoch(session, &ts);
```

Comment:
Note that this random function doesn't include WT-10338, which adds the thread ID into the RNG.